packages/dcos-checks/extra/dcos-checks.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=DC/OS Poststart Checks: Run node checks
```

Comment:
Let's say `Run node-poststart checks` in the description, to distinguish from the eventual prestart systemd unit.